lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -71,6 +73,7 @@
                         ['topics/:topic_id/search(/)', _.bind(this.searchTeams, this)],
                         ['topics/:topic_id/create-team(/)', _.bind(this.newTeam, this)],
                         ['topics/:topic_id/:team_id/edit-team(/)', _.bind(this.editTeam, this)],
+                        ['topics/:topic_id/:team_id/edit-team/manage-members(/)', _.bind(this.editTeamMembers, this)],
```

Comment:
Good point-- I copied what was there for edit-team, but they both seem wrong. I will change them both.